airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -876,12 +876,10 @@ def execute(self, context: Context) -> None:
                     project_id=self.project_id,
                     region=self.region,
                     cluster_name=self.cluster_name,
-                    request_id=self.request_id,
-                    retry=self.retry,
                     end_time=end_time,
                     metadata=self.metadata,
                     impersonation_chain=self.impersonation_chain,
-                    polling_interval=self.polling_interval_seconds,
+                    polling_interval_seconds=self.polling_interval_seconds,
```

Comment:
Just one question - I guess this is in trigger so there are no worries about back-compatibility ?

Reply:
yes, I think that is correct.